airbyte-integrations/connectors/source-orb/bootstrap.md

```diff
@@ -0,0 +1,22 @@
+## Streams
+
+Orb is a REST API. Connector has the following streams, and all of them support incremental refresh.
+
+* [Subscriptions]( https://docs.withorb.com/reference/list-subscriptions)
+* [Plans](https://docs.withorb.com/reference/list-plans)
+* [Customers](https://docs.withorb.com/reference/list-customers) 
+* [Credits Ledger Entries](https://docs.withorb.com/reference/view-credits-ledger)
+
+Note that the Credits Ledger Entries must read all Customers for an incremental sync, but will only incrementally return new ledger entries for each customer.
+
+Since the Orb API does not allow querying objects based on `updated_at`, these incremental syncs will capture updates to newly created objects but not resources updated after object creation.
```

Comment:
How often do we expect "updates" on objects after creation?

How do we deal with those "rare" situations if that happens? Should the connector do a full refresh occasionally to verify that we are not missing such updates at a lower frequency?

Reply:
This will depend on the resource: 

I expect the `Subscription` and `Customer` resources will indeed change (e.g. the state of a subscription may become inactive, or a Customer may have a new shipping address) often. On the other hand, Credit Ledger Entries are immutable and I do not expect the `Plan` resource to change often either.

I'll go ahead and document that in this `boostrap.md` file!

As you said, I would expect there to be a second connection that occasionally runs to resync all data. The other option is for me to implement a "lookback" feature similar to the Stripe Connector (which features a `lookback_window_days` to always resync _n_ days in the past). Do you recommend adding that parameter, or is it common to have two connections in this situation?

Reply:
It's probably better to implement the `lookback` feature as you described directly in the connector's option. 

So, this limitation is more obvious to the user when configuring this connector rather than expecting them to know and guess they need a second full refresh connection for such cases.